lib/kubernetes-deploy/kubeclient_builder.rb

```diff
@@ -100,17 +96,12 @@ def build_networking_v1_kubeclient(context)
       )
     end
 
-    def config_files
-      # Split the list by colon for Linux and Mac, and semicolon for Windows.
-      kubeconfig.split(/[:;]/).map!(&:strip).reject(&:empty?)
-    end
-
     def validate_config_files
       errors = []
-      if config_files.empty?
+      if @kubeconfig_files.empty?
         errors << "Kube config file name(s) not set in $KUBECONFIG"
       else
-        config_files.each do |f|
+        @kubeconfig_files.each do |f|
           unless File.file?(f)
```

Comment:
This might be nicer as a one-liner
`errors << "Kube config not found at ${f}" unless File.file(f)`